libs/design-system/web-component-card/src/lib/web-component-card.component.scss

```diff
@@ -0,0 +1,59 @@
+:host {
+  display: block;
+  position: relative;
+  border-radius: 0.5rem;
+  overflow: hidden;
+  background-color: var(--sys-surface-container-low);
+  box-shadow: 0px 5px 16px 0px rgba(from var(--sys-shadow) r g b / 0.24);
+
+  .image {
+    height: 15rem;
+    img {
+      height: 15rem;
+      width: 100%;
+    }
+  }
+
+  .cardContent {
+    padding: 1.5rem 1.5rem 0 1.5rem;
+  }
+
+  .default_style {
+    font: var(--sys-title-large);
```

Comment:
Please set letter-spacing along with the font. `letter-spacing: var(--sys-title-large-tracking)`